packages/lib/modules/pool/PoolDetail/PoolActivity/usePoolActivity.tsx

```diff
@@ -41,7 +41,7 @@ function _usePoolActivity() {
   const [isExpanded, setIsExpanded] = useState(false)
   const { isChartView } = usePoolActivityViewType()
 
-  const _chain = slugToChainMap[chain as ChainSlug]
+  const _chain = getChainSlug(chain as ChainSlug)
```

Comment:
The type of chain here is `string | string[]`
Is that cast safe (not familiar with this code).